backend/src/__tests__/quizCreation.test.ts

```diff
@@ -36,6 +42,39 @@ const wrongMetaTypes = {
 	questions: correct.questions,
 };
 
+const metaInjection = {
+	meta: {
+		title: correct.meta.title,
+		author: correct.meta.author,
+		pointDefault: 15,
+		timeDefault: 10,
+		kenobi: "I am old Lol",
+	},
+	questions: correct.questions,
+};
+
+const injectedQuestion = {
+	questionText: 'Are we human?',
+	answerTexts: [
+		'Yes',
+		'No',
+		'NONONONONOWAITWAITWAIT',
+		'idunno man I just work here',
+	],
+	correctAnswers: [1],
+	time: 10,
+	anakin: "you underestimate my power.",
+};
+
+const injectedList = correct.questions;
+injectedList.push(injectedQuestion);
+
+const questionInjectionQuiz = {
+	meta: correct.meta,
+	questions: injectedList,
+	injection: 'I have the high ground!',
+};
+
```

Comment:
Added these tests for ease of testing injection.

Reply:
you're right, i forgot about that

Reply:
It's alright, I didn't include these tests from the start.
Everything else is great from what I see, good work!